src/integration-test/java/com/commercetools/sync/integration/commons/utils/CartDiscountITUtils.java

```diff
@@ -138,7 +138,7 @@ public static void populateTargetProject() {
         int counter = 0;
         int current = 1;
 
-        while (counter != 100) {
+        while (counter != capacity) {
```

Comment:
is this change related to this PR?

Reply:
here I am passing capacity as a parameter but not using it :) I realized this problem when I needed to create sort orders more than 100 :)

Reply:
698c1f5